test/ui-automation/test/specs/credential-interaction-flow.js

```diff
@@ -124,7 +124,7 @@ credential.set("VaccinationCertificate", {
       id: "did:key:zUC72c7u4BYVmfYinDceXkNAwzPEyuEE23kUmJDjLy8495KH3pjLwFhae1Fww9qxxRdLnS2VNNwni6W3KbYZKsicDtiNNEp76fYWR6HCD8jAz6ihwmLRjcHH6kB294Xfg1SL1qQ",
       name: "didkey-bbsblssignature2020-bls12381g2",
     },
-    name: "COVID-19 Vaccination Certificate",
+    name: "COVID19 Vaccination Certificate",
```

Comment:
Why do we need this change ?

Reply:
When we try to rename credential names in the loop since '-' is not accepted character 'COVID-19' it throws validation error.

Reply:
We can skip updating in this as we are already testing the feature in WACI test files.